action.yml

```diff
@@ -25,9 +29,10 @@ runs:
       run: |
         echo "AWS_CLI_VERSION=${{ inputs.version }}" >> $GITHUB_ENV
         echo "VERBOSE=${{ inputs.verbose }}" >> $GITHUB_ENV
+        echo "LIGHTSAILCTL=${{ inputs.lightsailctl }}" >> $GITHUB_ENV
       shell: bash
     - id: install-aws-cli
-      run: sudo ${{ github.action_path }}/entrypoint.sh $AWS_CLI_VERSION
+      run: sudo --preserve-env ${{ github.action_path }}/entrypoint.sh
```

Comment:
Today I Learned about the z--preserve-env` flag :) Thanks

Reply:
Oddly enough, me too! Couldn't understand why it wasn't passing my values through until I stuck a call to `env` at the top of the script. :D